### ISSUE TITLE
How many gpus was used when running the experiments? This can affect results especially with batchnorm + torch.DataParallel

### ISSUE THREAD

**Issue by Phuoc-Hoan-Le** [liuzechun/ReActNet#33]
I am not getting the exact same results as stated in the paper, most likely, due to using multi-gpu in torch.nn.dataparallel which can lead to different results depending the number of gpus used.

I was wondering how many gpus was used for your experiments during training? Have you even tried running more than one gpu to see how much would it affect the results?

**Comment by Phuoc-Hoan-Le** [comment]
@szq0214 @liuzechun Any update yet?
